docs/user/mongodb.md

```diff
@@ -0,0 +1,264 @@
+# MongoDB setup tutorial
+
+`autoplex` and `atomate2` use the MongoDB database framework to store the data output from the calculations. 
+The data is stored in a JSON-like format which makes it easier to access through Python for further post-processing.
+
+## MongoDB setup
+
+MongoDB is best run on the back-end. For this, please ask your IT department to install [MongoDB](https://www.mongodb.com/) for you!
+There might be cases, where this is not feasible and then you need to run a mongodb on the FRONT-end yourself.
+This guide will walk you through the steps that you to do for running a mongodb in front-end.
+
+  * Get mongodb from here: https://www.mongodb.com/try/download/community
+  * You can e.g. download "Platform: SUSE 15" and "Package: tgz" and put it on your remote cluster. Check the platform-dependency of your cluster!!!
+  * Extract the files from the downloaded archive and add the executables within to your PATH environment variable i.e open the  `~/.bashrc` file and add following line to it.
+    ```bash
+    export PATH="/path/to/the/mongodb-directory/bin:$PATH"
+    ``` 
+  * Now, you can start the mongodb database with the following command:
+    ```bash
+    mongod --bind_ip_all  --dbpath /this_is_the_path_to_your_db --quiet --port 27017
```

Comment:
I think this executable is not available in the recent versions. Can anyone check it please?

Reply:
you mean with the most recent MongoDB version?

Reply:
any version above 5 I think

Reply:
Hmmm... Maybe I mention the version?

Reply:
I would say this will work for other versions too: https://www.mongodb.com/docs/manual/tutorial/manage-mongodb-processes/

Reply:
Issue was mongo shell https://www.mongodb.com/docs/manual/reference/mongo/ and I will mention this in the doc :)